File: VB/Task 11.vb
Module Module1

    Sub Main()
        Console.WriteLine("Enter String: ")
        Dim input As String = LCase(Console.ReadLine())

        For count = 1 To Len(input)
            For count2 = 1 To Len(input)
                If Mid(input, count, 3) = "not" And Mid(input, count2, 3) = "bad" Then
                    Dim charsToReplace As Integer = (count2 - count)
                    input = Left(input, (count - 1)) & "good" & Mid(input, (count2 + 3), (Len(input)))
                End If
            Next
        Next

        Console.WriteLine(input)

        Console.ReadLine()
    End Sub

End Module

File: Python/Task_11.py
String = input("Enter String: ").lower()

for x in range(0, len(String)):
    for y in range(0, len(String)):
        if String[x:x+3] == "not" and String[y:y+3] == "bad":
            String = String[:x] + "good" + String[y+3:]

print(String)